Mention how to use the `RuntimeHostConfigurationOption` MSBuild item (#30003)
dotnet/docs: docs/core/runtime-config/index.md

      "version": "3.1.0"
    },
    "configProperties": {

>       "System.Drawing.EnableUnixSupport": true,

Reviewer: I thought that support for System.Drawing on Unix was dropped. I cannot find a statement on that.

@terrajobst

Author: Yes, it will be dropped completely in .NET 7, see [Make System.Drawing.Common only supported on Windows](https://github.com/dotnet/designs/blob/main/accepted/2021/system-drawing-win-only/system-drawing-win-only.md). On .NET 6, the `System.Drawing.EnableUnixSupport` runtime configuration switch enables `System.Drawing.Common` to work on non-Windows platforms. I just used `System.Drawing.EnableUnixSupport` to illustrate how to use the `RuntimeHostConfigurationOption` MSBuild item for runtime configuration switches that don't have a specific MSBuild property.  Granted, that's probably not the best example since this particular runtime switch will not have any effect once .NET 7 is released!

Maybe we should find a better runtime configuration switch example for this doc? Or maybe the `System.Net.DisableIPv6` example is enough?

Reviewer: Got it. That clarifies things. Yes, that's the issue I was looking for. Thanks for that!

Ya, I'd stick with options that have a longer shelf life.

Author: I picked another runtime switch in b4961c41bdaf7a1a713b821a2f8b708201958f54: `System.Globalization.UseNls` instead of `System.Drawing.EnableUnixSupport`. It was a tough one since many runtime switches have a documented MSBuild property in the (not very discoverable) [Libraries Feature Switches](https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/feature-switches.md) documentation.

    <ThreadPoolMaxThreads>25</ThreadPoolMaxThreads>
  </PropertyGroup>

  <ItemGroup>
    <RuntimeHostConfigurationOption Include="System.Drawing.EnableUnixSupport" Value="true" />
    <RuntimeHostConfigurationOption Include="System.Net.DisableIPv6" Value="true" />
  </ItemGroup>

</Project>
```

> MSBuild properties for configuring run-time behavior are noted in the individual articles for each area, for example, [garbage collection](garbage-collector.md). They are also listed in the [Runtime configuration](../project-sdk/msbuild-props.md#runtime-configuration-properties) section of the MSBuild properties reference for SDK-style projects.

Reviewer 2: ```suggestion
MSBuild properties for configuring the behavior of the runtime are noted in the individual articles for each area, for example, [garbage collection](garbage-collector.md). They are also listed in the [Runtime configuration](../project-sdk/msbuild-props.md#runtime-configuration-properties) section of the MSBuild properties reference for SDK-style projects.
```

Reviewer 2: I know it's unrelated to your changes, but since we're in the file...

> For runtime configuration settings that don't have a specific MSBuild property, the `RuntimeHostConfigurationOption` MSBuild item may be used instead.

Reviewer 2: ```suggestion
For runtime configuration settings that don't have a specific MSBuild property, you can use the `RuntimeHostConfigurationOption` MSBuild item instead.
```

Reviewer 2: What do you think about moving this sentence to insert it at line 92?

Author: Good idea, done in dfb2cd
